javascript/builtins/Intl.json

```diff
@@ -46,7 +46,7 @@
               "version_added": true
             },
             "webview_android": {
-              "version_added": false
+              "version_added": "37"
```

Comment:
Since `webview_android` is based off of `chrome_android`, and `chrome_android` had support for them, I picked the earliest `webview_android` version that was available and was Chrome-based.  Since this doesn't actually violate any version consistency tests, however (reverting the file doesn't raise linter errors), I'll be reverting these changes!

Reply:
Since `webview_android` is based off of `chrome_android`, and `chrome_android` had support for them, I picked the earliest `webview_android` version that was available and was Chrome-based.  I'm quite certain that the JavaScript APIs are the same across both, though I could be wrong?  (Not an Android user myself.  😛)

With the version of the file that's currently in the `master` branch, the linter test raises the following errors:

```
  Found 4 inconsistent feature(s) in javascript/builtins/Intl.json:
  → 1 × Intl [javascript.builtins.Intl]: 
    → No support in webview_android, but this is not declared for sub-feature(s): NumberFormat, PluralRules, RelativeTimeFormat
  → 1 × Collator [javascript.builtins.Intl.Collator]: 
    → No support in webview_android, but this is not declared for sub-feature(s): caseFirst
  → 1 × DateTimeFormat [javascript.builtins.Intl.DateTimeFormat]: 
    → No support in webview_android, but this is not declared for sub-feature(s): iana_time_zone_names, hourCycle, formatToParts
  → 1 × resolvedOptions [javascript.builtins.Intl.DateTimeFormat.resolvedOptions]: 
    → No support in webview_android, but this is not declared for sub-feature(s): computed_timezone
```

Reply:
Okay, I see, this makes some sense now. I believe the changes to 37 are then correct in this PR. Marking them as resolved.